cache/year_run_stats.py

```diff
@@ -0,0 +1,53 @@
+from cache.cache_helpers import RunStats
+from sts_profile import get_profile
+
+_run_stats = RunStats()
+
+def update_run_stats():
+    # we should only have to load this one time this way, after that we can just use the most recent run to update values
+    runs = list(get_profile(0).runs) # BaalorA20 profile
+    if not _run_stats.streaks.is_loaded:
+        _run_stats.streaks.all_character_count = 0
+
+        for run in runs[1:]:
```

Comment:
This should probably be something like `reversed` here, so that it starts at the beginning. And then you probably don't need the second block for just that one. Although I'm not sure if you had some special handling there.

Reply:
The second block is just so it gets run whenever it syncs. The rest of the loop happens one time only.